docs/bigquery-usage.rst

```diff
@@ -34,31 +34,74 @@ Authorization / Configuration
      >>> client = bigquery.Client()
 
 - Override the credentials inferred from the environment by passing explicit
-  ``credentials`` to one of the alternative classmethods, `from_service_account_json`:
+  ``credentials`` to one of the alternative ``classmethod`` factories,
+  `:meth:gcloud.bigquery.client.Client.from_service_account_json`:
 
   .. doctest::
 
      >>> from gcloud import bigquery
      >>> client = bigquery.Client.from_service_account_json('/path/to/creds.json')
 
-  or `from_service_account_p12`:
+  or `:meth:gcloud.bigquery.client.Client.from_service_account_p12`:
 
   .. doctest::
 
      >>> from gcloud import bigquery
      >>> client = bigquery.Client.from_service_account_p12('/path/to/creds.p12', 'jrandom@example.com')
 
-- Override the project inferred from the environment by passing an explicit
-  ``project`` to the constructor, or to either of the alternative classmethods:
+
+Projects
```

Comment:
OK cool. When testing the [Resource Manager API][1] I got back 18 projects but when using [`storage.projects.list`][2] I only got back two.

They were `danielhermes-gcloud-research` and `gcloud-python-regression-tests` (my only projects with BigQuery enabled).

[1]: https://developers.google.com/apis-explorer/#p/cloudresourcemanager/v1beta1/cloudresourcemanager.projects.list
[2]: https://developers.google.com/apis-explorer/#p/bigquery/v2/bigquery.projects.list